client/src/components/RelatedItems/Modal.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Overlay = styled.div`
+position: fixed;
+left:0;
+top:0;
+right:0;
+bottom:0;
+background-color: rgba(0, 0, 0, 0.2);
+display: flex;
+align-items:center;
+justify-content: center;
+`;
+
+const Content = styled.div`
+  position: fixed;
+  left:400px;
+  top:200px;
+  right:400px;
+  bottom:200px;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  align-items:center;
+  justify-content: center;
+`;
+
+const Body = styled.div`
+padding: 10px;
```

Comment:
same thing here

Reply:
thanks for reviewing, will tap those out for sure